### DISCUSSION TITLE
Bulk sell crops

### DISCUSSION THREAD

**Discussion by AzaroQL** [sunflower-land/sunflower-land#1175] [Ideas]
Issue:
Sometimes I want to sell biggest part of my crops, but not all. 

Current way:
Sell crops one by one till desired amount is left. Can be problematic if you have hundreds of crops.

Possible fix:
When you click 'sell all' you see this popup:
![image](https://user-images.githubusercontent.com/9656961/177095663-10e71480-f722-48fb-abd8-8bf7eca51244.png)

Number in the dialog can be converted to input field, so if someone wants to sell only part of his crops he will edit the number and click sell.
For normal flow with 'sell all' nothing changes, max amount of crops is default input value.

In my opinion its the easiest solution to this problem which doesnt complicate UI and does the job.

**Comment by Demonbane18** [comment]
a Sell 10 and sell 100 button is fine as well but its better if we do have a specific quantity sell and specific quantity buy for crops

**Comment by veenhdek** [comment]
I like this idea, have really run into this personally a lot while saving crops for a purpose (cakes, base amount to maintain) while also selling for SFL to feed my resource mining

**Comment by hujikol** [comment]
Currently, I've built it before (#1090 ) and got rejected cause the UI is not fit in the mobile version.

For custom input, it's already been suggested by many PR before and it goes rejected because the team is not into it at the moment (probably because they are busy building the market).
Here are some PR:  #703 (best UIX in this PR is this *i think [here](https://github.com/sunflower-land/sunflower-land/pull/703#issuecomment-1114125509)), #844 , (mine) #1090

I don't know what the team aims to build right now when there is already a custom input format in a donation to the farm or team that can be used again.
<img src="https://user-images.githubusercontent.com/12151051/178134442-3e26be15-9441-4232-9b4d-490c5e66fb08.png" width="360" />


or in the goblin bank withdraw.
<img src="https://user-images.githubusercontent.com/12151051/178134505-d5469cfc-ffeb-43b5-8392-5b0d917b320e.png" width="360" />

Sorry to tag you here sir @spencerdezartsmith  following your suggestion that adding 'Sell 10' to align with the 'Buy 10', in [here](https://github.com/sunflower-land/sunflower-land/pull/1090#issuecomment-1158325445), I've found a problem with, 

- when the crops go down to 10 we will have 2 buttons doing the same job, 'Sell All' will be the same as 'Sell 10', 
> for that problem I don't know if the 'Sell 10' could be disabled or completely not in the display.
<img src="https://user-images.githubusercontent.com/12151051/178135839-0063259c-c8ab-425e-9f80-a6e3e3493722.png" width="360" />

https://user-images.githubusercontent.com/12151051/178136507-ee72d4ca-3b93-4cb3-9f71-0c58d9503489.mp4

Edit:
It will be a great help from the community if we can also think together about the UX of this problem within selling and buying in bulk.

**Reply by hujikol** [reply]
and another same issue will occur when adding the 'Sell 100' button

**Reply by hujikol** [reply]
and also im planning to add buy all button with confirmation pop out
